byway/pom.xml

```diff
@@ -43,13 +43,6 @@
       <version>1.9.59</version>
     </dependency>
 
-    <!-- https://mvnrepository.com/artifact/com.google.flogger/flogger -->
-    <dependency>
-      <groupId>com.google.flogger</groupId>
-      <artifactId>flogger</artifactId>
-      <version>0.5.1</version>
-    </dependency>
-
     <!-- https://mvnrepository.com/artifact/com.google.flogger/flogger -->
     <dependency>
```

Comment:
Please check this file out back to the version in master.

Reply:
I checked out pom.xml and the java files from master.